### PR TITLE
Introduced FileFactory::createFromUri method

### PR DESCRIPTION
**Pull request by szymach**
Solves #42

### CODE REVIEW

File: src/Integration/FlySystem/Upload/FileFactory.php [hunk 1]
```diff
@@ -48,28 +72,90 @@ public function createFromPath(string $path, ?string $targetName = null): Files\
...
         $size = $stream->getSize();
         if (null === $size) {
-            throw new RuntimeException("Unable to read size of file at path \"{$path}\".");
+            throw new RuntimeException(
+                sprintf('Unable to file size of "%s".', (string) $uri)
+            );
         }
 
-        return new FlySystem\UploadedWebFile(
+        $mimeType = $this->getMimeTypeFromFileContents($stream->getContents());
```
Comment (chives) [inline src/Integration/FlySystem/Upload/FileFactory.php:119]: why not try to extract `mime-type` from `content-type` response header first?

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Review by chives — APPROVED** [review]

**Review by rn0 — APPROVED** [review]

**Pull request merged**